lib/utils/boot.html

```diff
@@ -18,6 +21,12 @@
   window.Polymer = function(info) {
     return window.Polymer._polymerFn(info);
   }
+
+  // support user settings on the Polymer object
+  if (userPolymer) {
+    Object.assign(Polymer, userPolymer);
+  }
```

Comment:
Just noting, this fixes the ability to set `Polymer.sanitizeDOMValue` before loading polymer, closing a loophole where it's conceivable that main-document e.g. `dom-repeat`'s could upgrade before the sanitization hook is installed. 👍